### PR TITLE
#6 Created Project page list,and created skeleton of component projec…

### PR DESCRIPTION
**Pull request by tomaszprc**
…t details page

### CODE REVIEW

File: src/pages/Projects/Projects.tsx [hunk 1]
```diff
@@ -1,9 +1,48 @@
...
+        {projects.map((project) => (
+          <ProjectElement
+            id={project.id}
+            title={project.title}
+            description={project.description}
+            timeTrackIds={project.timeTrackIds}
+          />
+        ))}
+      </div>
+    </Container>
```
Comment (korczas) [inline src/pages/Projects/Projects.tsx:44]: .container class implements max-width and width for the whole app. You already used this on top level, so there is no sense to duplicating that in childrens. 

If you need, you can add extra class to center content with `margin: 0 auto`
If you need to wrap the content (react expect only one child in return function) you can use `<Fragment>...</Fragment>` or shortcut for fragment:  `<>....</>`

Please fix that also on other components (eg. Navbar)

File: src/styles/components/_project-element.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    &--remove {
+      background-color: #e93c30;
+    }
+
+    &--edit {
+      background-color: #1f5aa6;
+    }
+  }
+}
```
Comment (korczas) [inline src/styles/components/_project-element.scss:28]: sort css properties

In selleo there are some ppl who take a lot of attention to keep css props sorted. 99.9% of time sorting don't impact on css output, so try to keep them sorted.

File: src/styles/components/_title.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.title {
+  display: flex;
+  justify-content: center;
+  font-size: $font-size-xl;
+  font-weight: bold;
```
Comment (korczas) [inline src/styles/components/_title.scss:5]: sort

File: src/components/ProjectElement/ProjectElement.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+interface ProjectElementProps {
+  id: Number;
+  title: String;
+  description: String;
+  timeTrackIds: Array<Number>;
```
[the remark this answers is not in the document]
Comment (tomaszprc) [reply src/components/ProjectElement/ProjectElement.tsx:7]: Fixed in master

File: src/components/Title/Title.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+interface TitleProps {
+  title: String;
```
[the remark this answers is not in the document]
Comment (tomaszprc) [reply src/components/Title/Title.tsx:2]: Fixed in master

File: src/pages/Projects/Projects.tsx [hunk 6]
```diff
@@ -1,9 +1,48 @@
...
 
 interface ProjectsProps {}
 
-const Projects: FunctionComponent<ProjectsProps> = ({}) => {
-  return <div>Projects</div>;
+interface ProjectsState {
+  id: Number;
+  title: String;
+  description: String;
+  timeTrackIds: Array<Number>;
```
[the remark this answers is not in the document]
Comment (tomaszprc) [reply src/pages/Projects/Projects.tsx:12]: fixed in master

**Pull request merged**